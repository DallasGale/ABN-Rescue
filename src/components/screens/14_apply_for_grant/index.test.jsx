/* eslint-env jest */

import React from 'react'
import { render } from '@testing-library/react'

import ApplyForGrantFork from '.'

describe('<ApplyForGrantFork />', () => {
  it('renders without crashing', () => {
    const component = render(<ApplyForGrantFork />)
    expect(component).toMatchSnapshot()
  })
})
