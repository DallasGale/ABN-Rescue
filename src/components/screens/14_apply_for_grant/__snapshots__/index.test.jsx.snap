// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ApplyForGrantFork /> renders without crashing 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="styled__StyledContentContainer-sc-181bkbc-1 eafeFg"
      >
        <h2
          class="styled__StyledHeading-sc-181bkbc-4 ixZsfG"
        >
          Kickstart your ambition with 
          <br />
          a business grant worth $40k
        </h2>
        <div
          class="styled__StyledContentWrapper-sc-181bkbc-2 dEXGnT"
        >
          <div
            class="styled__StyledContent-sc-181bkbc-3 ereoUM"
          >
            <p
              class="typography__StyledParagraph-sc-1n1kkhl-2 fPrFsI"
              font-size="1.25rem"
            >
              Have a bold business idea that needs an ambition boost? Then CGU are here to help. If you're a start up, scale up or anything in between, apply for a CGU Ambition Grant today to be in the running to receive a business-changing grant worth $40,000.
            </p>
            <p
              class="typography__StyledParagraph-sc-1n1kkhl-2 kJErQP"
            >
              Your ambition means a lot to us so, take your time to respond and follow these simple steps to apply:
            </p>
            <p
              class="typography__StyledParagraph-sc-1n1kkhl-2 eqWrYx"
            >
              <strong>
                To apply, follow these simple steps:
              </strong>
            </p>
            <ol
              class="styled__StyledOl-sc-181bkbc-5 dhuwfv"
            >
              <li>
                Download the application form (see link below)
              </li>
              <li>
                Take your time to complete the form and understand the
                 
                <a
                  class="styled__StyledLink-sc-181bkbc-6 kXWLUp"
                  href="/abn-rescue/grant-application-terms-and-conditions.pdf"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  terms & conditions
                </a>
              </li>
              <li>
                Return to this page and submit your application form by Tuesday, 15th October
              </li>
            </ol>
            <a
              class="styled__StyledCtaLink-sc-181bkbc-9 hYHNDQ"
              data-tracking="btn-click:downloadApplicationForm"
              href="static/media/CGU-AMBITION-GRANT-APPLICATION-FORM.pdf"
              role="button"
              target="_blank"
            >
              <span>
                Download application form
              </span>
              <svg
                height="14"
                width="44"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill="none"
                  fill-rule="evenodd"
                  stroke="#090D2B"
                  stroke-width="2"
                >
                  <path
                    d="M0 7h42.232M36.379 12.657L42.036 7l-5.657-5.657"
                  />
                </g>
              </svg>
            </a>
          </div>
          <div
            class="styled__StyledCtaSection-sc-181bkbc-7 jRngRb"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="styled__StyledContentContainer-sc-181bkbc-1 eafeFg"
    >
      <h2
        class="styled__StyledHeading-sc-181bkbc-4 ixZsfG"
      >
        Kickstart your ambition with 
        <br />
        a business grant worth $40k
      </h2>
      <div
        class="styled__StyledContentWrapper-sc-181bkbc-2 dEXGnT"
      >
        <div
          class="styled__StyledContent-sc-181bkbc-3 ereoUM"
        >
          <p
            class="typography__StyledParagraph-sc-1n1kkhl-2 fPrFsI"
            font-size="1.25rem"
          >
            Have a bold business idea that needs an ambition boost? Then CGU are here to help. If you're a start up, scale up or anything in between, apply for a CGU Ambition Grant today to be in the running to receive a business-changing grant worth $40,000.
          </p>
          <p
            class="typography__StyledParagraph-sc-1n1kkhl-2 kJErQP"
          >
            Your ambition means a lot to us so, take your time to respond and follow these simple steps to apply:
          </p>
          <p
            class="typography__StyledParagraph-sc-1n1kkhl-2 eqWrYx"
          >
            <strong>
              To apply, follow these simple steps:
            </strong>
          </p>
          <ol
            class="styled__StyledOl-sc-181bkbc-5 dhuwfv"
          >
            <li>
              Download the application form (see link below)
            </li>
            <li>
              Take your time to complete the form and understand the
               
              <a
                class="styled__StyledLink-sc-181bkbc-6 kXWLUp"
                href="/abn-rescue/grant-application-terms-and-conditions.pdf"
                rel="noopener noreferrer"
                target="_blank"
              >
                terms & conditions
              </a>
            </li>
            <li>
              Return to this page and submit your application form by Tuesday, 15th October
            </li>
          </ol>
          <a
            class="styled__StyledCtaLink-sc-181bkbc-9 hYHNDQ"
            data-tracking="btn-click:downloadApplicationForm"
            href="static/media/CGU-AMBITION-GRANT-APPLICATION-FORM.pdf"
            role="button"
            target="_blank"
          >
            <span>
              Download application form
            </span>
            <svg
              height="14"
              width="44"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                fill="none"
                fill-rule="evenodd"
                stroke="#090D2B"
                stroke-width="2"
              >
                <path
                  d="M0 7h42.232M36.379 12.657L42.036 7l-5.657-5.657"
                />
              </g>
            </svg>
          </a>
        </div>
        <div
          class="styled__StyledCtaSection-sc-181bkbc-7 jRngRb"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
